server/migrations/1582144372565-FixChapterRoleRelation.ts

```diff
@@ -8,6 +8,10 @@ export class FixChapterRoleRelation1582144372565 implements MigrationInterface {
       `ALTER TABLE "user_chapter_roles" DROP CONSTRAINT "FK_c4b23ce3d811d599cea97d064fc"`,
       undefined,
     );
+    await queryRunner.query(
+      `ALTER TABLE "user_chapter_roles" DROP COLUMN "chapter_id"`,
+      undefined,
+    );
     await queryRunner.query(
       `ALTER TABLE "user_chapter_roles" ADD "chapter_id" integer`,
       undefined,
```

Comment:
On lines 11-14 we are dropping `chapter_id` and on 15-17 we are adding it back. Any specific reasons for that?

Reply:
Yeah this was a dumb query, fixed it